### DISCUSSION TITLE
Selecting values in forms via Query String parameters

### DISCUSSION THREAD

**Discussion by privatemaker** [decaporg/decap-cms#7042] [Q&A] [answered]
I was curious about the idea of deep linking to forms in my Decap CMS with query string values and figuring out how to make various values be automatically selected (instead of requiring user action). 

For example instead of linking to:

```
https://example.org/admin/#/collections/work/new
```

Linking to the following

```
https://example.org/admin/#/collections/work/new?client=secret-fediverse-app
```

The goal being that the field with `name: client` would automatically select value (or populate the field) with the `value: secret-fediverse-app`

Upon testing this out of curiosity it actually works already 🥳🎈 

I tested with multiple values in the QS and in the following widget types: `select, relation, string`

Perhaps this is just missing from the documentation? Or I missed it in the documentation?

---

*NOTE:* One quirk which affects deep-linking in general (not just query string values) is the URL gets striped of the all path values except the root.

```
https://example.org/admin/
```

**Comment by martinjagodic** [comment]
@privatemaker how did you achieve this exactly?

**Comment by privatemaker** [comment]
Nothing special. Simply loading the URL *with* the Query String values:

```
https://example.org/admin/#/collections/work/new?client=secret-fediverse-app&date=2023-12-31
```
Once the URL is constructed it works either by clicking on it or manually loading it in the browser

**Reply by martinjagodic** [reply]
Nice, I did not know about this. Are you willing to contribute to the docs to explain this? A sentence or two will be enough.

**Reply by privatemaker** [reply]
Sure. I'll get to that shortly!

**Reply by privatemaker** [reply, marked as answer]
Lol 🤣 @martinjagodic this *is* already documented under Beta Features: [Dynamic Default Values](https://decapcms.org/docs/beta-features/#dynamic-default-values). That list is just so long and semantically covers almost all of the documentation- I must have read this ages ago and forgot.
 
Hence the need to move things out of `Beta` getting start on #7040 it is!

**Reply by martinjagodic** [reply]
Well, now we know 😁
